examples/rj9a/websocket_upgrade.clj

```diff
@@ -0,0 +1,62 @@
+(ns rj9a.websocket-upgrade
+  (:gen-class)
+  (:require [clojure.string :refer [lower-case]]
+            [ring.adapter.jetty9 :as jetty]))
+
+(defonce server (atom nil))
+
+(defn websocket-upgrade? [{:keys [headers]}]
+  (and (= "websocket" (lower-case (get headers "upgrade")))
+       (= "upgrade" (lower-case (get headers "connection")))))
+
+(defn websocket-upgrade-response [ws-handler]
+  {:status 101 ;; http 101 switching protocols
+   :headers {"upgrade" "websocket"
+             "connection" "upgrade"}
+   :ws ws-handler})
```

Comment:
We can make this a built-in helper function for upgrading to websocket connection

Reply:
Good call! I'll do that.

Reply:
I moved those two fns over to the websocket namespace with different names. Please let me know if you think they need better names or live in some other namespace.

Reply:
The names looks good to me. Thanks!